build-site.py

```diff
@@ -21,9 +24,17 @@
 if args.enus:
     langmsg = 'in en-US only.'
     languages = ['en-US']
+    calendar_locales = [{
+        'US': 'United States',
+    }]
 else:
     langmsg = 'in all languages.'
     languages = settings.PROD_LANGUAGES
+    calendar_locales = settings.CALENDAR_LOCALES
+
+#if args.buildcalendar:
```

Comment:
Seems like this needs to be uncommented for this patch to run :)

Also, I would actually like `--buildcalendar` to run by itself only, so put it in as an `elif`. The calendar build process will run on a long schedule(once a month or less) and we'll just check the files into the repository. Rebuilding with every website build would also massively increase build times.